### PR TITLE
fix atari examples

### LINKED ISSUE #205: small bug in example
- [ ] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [ ] RL algorithm bug
    + [ ] documentation request (i.e. "X is missing from the documentation.")
    + [ ] new feature request
- [x] I have visited the [source website](https://github.com/thu-ml/tianshou/)
- [x] I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
- [x] I have mentioned version numbers, operating system and environment, where applicable:
  ```python
   >>> import tianshou, torch, sys
   >>> print(tianshou.__version__, torch.__version__, sys.version, sys.platform)
   0.2.6 1.6.0 3.8.5 (default, Sep  4 2020, 07:30:14) 
   [GCC 7.3.0] linux

  ```

The atari_dqn .py in '/example' is something wrong. The ReplayBuffer is initialled by `buffer = ReplayBuffer(args.buffer_size, ignore_obs_next=True, save_last_obs=True, stack_num=args.frames_stack)`. But in the define is:
```python
class ReplayBuffer:
    def __init__(self, size: int, stack_num: int = 1,
                 ignore_obs_next: bool = False,
                 save_only_last_obs: bool = False,
                 sample_avail: bool = False) -> None:
        super().__init__()
```
Thanks

### PR DESCRIPTION
**Pull request by Trinkle23897**

### CODE REVIEW

**Review by Leeqh666 — APPROVED** [review]
Thanks

**Review by Leeqh666 — APPROVED** [review]

**Comment by Trinkle23897** [comment]
I'm waiting for another maintainer's approval.

**Review by youkaichao — APPROVED** [review]

**Pull request merged**
